File: app/src/main/java/com/github/premnirmal/ticker/ui/SpacingDecoration.java
package com.github.premnirmal.ticker.ui;

import android.graphics.Rect;
import android.support.v7.widget.RecyclerView;
import android.view.View;

/**
 * Created by premnirmal on 2/10/16.
 */
public class SpacingDecoration extends RecyclerView.ItemDecoration {

    private final int spacing;

    public SpacingDecoration(int spacing) {
        this.spacing = spacing;
    }

    @Override
    public void getItemOffsets(Rect outRect, View view, RecyclerView parent, RecyclerView.State state) {
        super.getItemOffsets(outRect, view, parent, state);
        outRect.bottom = spacing;
        outRect.top = spacing;
        outRect.left = spacing;
        outRect.right = spacing;
    }
}

File: app/src/main/kotlin/com/github/premnirmal/ticker/ui/SpacingDecoration.kt
package com.github.premnirmal.ticker.ui

import android.graphics.Rect
import android.view.View
import androidx.recyclerview.widget.RecyclerView

/**
 * Created by premnirmal on 2/27/16.
 */
open class SpacingDecoration(private val spacing: Int) : RecyclerView.ItemDecoration() {

  override fun getItemOffsets(
      outRect: Rect,
      view: View,
      parent: RecyclerView,
      state: RecyclerView.State
  ) {
    super.getItemOffsets(outRect, view, parent, state)
    outRect.bottom = spacing
    outRect.top = spacing
    outRect.left = spacing
    outRect.right = spacing
    outRect.bottom = spacing
  }
}